packages/core/src/document.ts

```diff
@@ -652,11 +645,8 @@ class Document extends EventEmitter implements DocStateHolder {
    */
   static async loadSchemaById<T extends Doctype>(context: Context, schemaDocId: string): Promise<T> {
     if (schemaDocId) {
-      const schemaDocIdParsed = DocID.fromString(schemaDocId)
-      if (!schemaDocIdParsed.commit) {
-        throw new Error("Commit missing when loading schema document")
```

Comment:
Can we keep this error around, which is a bit more descriptive than the `No commit information provided` message you'll get without it?  It's useful to know that we're in the context of loading a schema and that schemas must always be set at a specific commit

Reply:
OK